### PR TITLE
Add twist-stamper node so nav2 example works with humble

### PR DESCRIPTION
**Pull request by srmainwaring**
The main addition is to add a twist-stamper node to allow the nav2 example to work with ROS 2 Humble.

See https://github.com/ArduPilot/ardupilot_ros/pull/11#issuecomment-1934765581 for details. 

Other changes include:

- Update robot base footprint radius.
- Ensure the `SDF_PATH` env variable is set in launch file to allow robot state publisher to resolve the model (and so TF).
- Display robot model and odometry in rviz.

## Dependences

- https://github.com/ArduPilot/ardupilot_ros/pull/11

### CODE REVIEW

**Comment by Jagadeesh-pradhani** [comment]
Every thing works fine,
Just one thing that, twist_stamped node wasn't working when i ran it with navigation.
Later running it separately worked for me.

File: launch/twist_stamper.launch.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import os
+
```
Comment (Ryanf55) [inline launch/twist_stamper.launch.py]:
```suggestion
```

Unused import

**Review by Ryanf55 — APPROVED** [review]
The twist stamper is working as intended. Excellent work.

**Comment by srmainwaring** [comment]
Thanks @Ryan, I'll merge once the issue with the failing test is resolved.

Ah, that's an upstream issue with ardupilot_gazebo - missing the gstreamer deps in the cmake. Let me get onto that.

File: launch/navigation.launch.py [hunk 2]
```diff
@@ -40,6 +41,20 @@ def generate_launch_description():
...
+        package="twist_stamper",
+        executable="twist_stamper",
+        parameters=[
+            {"frame_id": "base_link"},
+        ],
+        remappings=[
+            ("cmd_vel_in", "cmd_vel"),
+            ("cmd_vel_out", "ap/cmd_vel"),
+        ],
+    )
```
Comment (pedro-fuoco) [inline launch/navigation.launch.py:56]: I see, that's a clever and organized way to work around the `nav2` twist situation, good job @srmainwaring!

**Pull request merged**